test/helpers/test_unit_utils.go

```diff
@@ -3,7 +3,7 @@
 // This source code is licensed under the Apache License, Version 2.0 license found in the
```

Comment:
can we rename this file to maybe `test_data.go`? And maybe move the functions that are creating & deleting files & directories to another file called `os_utils.go`.